### ISSUE TITLE
WT32-ETH01 board to support Openevse ethernet

### ISSUE THREAD

**Issue by jdgarcia99** [OpenEVSE/openevse_esp32_firmware#741]
I've been able to run openevse wifi gateway on ethernet with the wt32-eth01 board, which is way cheaper than the olimex one.

The platform env configuration for this board is as follows:

```
[env:wt32-eth01]
# For Wireless Tag
board = wt32-eth01
build_flags =
  ${common.build_flags}
  ${common.src_build_flags}
  -D WIFI_LED=12
  -D WIFI_LED_ON_STATE=HIGH
  -D WIFI_BUTTON=4
  -D WIFI_BUTTON_PRESSED_STATE=LOW
  -D RAPI_PORT=Serial2
  -D ENABLE_WIRED_ETHERNET
  -D RESET_ETH_PHY_ON_BOOT=1
  -D RX2=5
  -D TX2=17
  -D ETH_PHY_TYPE=ETH_PHY_LAN8720
  -D ETH_PHY_ADDR=1
  -D ETH_PHY_MDC=23
  -D ETH_PHY_MDIO=18
  -D ETH_CLK_MODE=ETH_CLOCK_GPIO0_IN
  -D ETH_PHY_POWER=16
```

This board is a little bit tricky with the boot process due to clock crystal, so I had to put a 1uF ceramic capacitor between pins EN and GND.

With this workaround the Board works flawlessly

Could you check and add as a new board support?

**Comment by jeremypoulter** [comment]
Great, would you be able to submit a PR?

**Comment by jdgarcia99** [comment]
Done!

**Comment by jdgarcia99** [comment]
> Great, would you be able to submit a PR?
> 

Have you been able to test it?

**Comment by jeremypoulter** [comment]
Closed via #743
